buildenv/jenkins/common/build

```diff
@@ -261,10 +261,28 @@ def build() {
 def archive() {
     stage('Archive') {
         timestamps {
-            sh "tar -zcvf ${TEST_PREFIX}`git -C openj9 rev-parse --short HEAD`${TEST_SUFFIX} openj9/test/"
-            archiveArtifacts artifacts: "**/${TEST_PREFIX}*${TEST_SUFFIX}", fingerprint: true, onlyIfSuccessful: true
-            sh "tar -C build/$RELEASE/images/ -zcvf ${SDK_PREFIX}`date +%Y%d%m%H%M`${SDK_SUFFIX} ${JDK_FOLDER}"
-            archiveArtifacts artifacts: "**/${SDK_PREFIX}*${SDK_SUFFIX}", fingerprint: true, onlyIfSuccessful: true
+              sh "tar -C build/${RELEASE}/images/ -zcvf ${SDK_FILENAME} ${JDK_FOLDER}"
+              if (ARTIFACTORY_SERVER) {
+                def server = Artifactory.server ARTIFACTORY_SERVER
+                // if the archive location is modified the definition of CUSTOMIZED_SDK_URL will also need to be updated
+                def uploadSpec = """{
+                  "files":[
+                    {
+                      "pattern": "${env.WORKSPACE}/${SDK_FILENAME}",
+                      "target": "${ARTIFACTORY_REPO}/${env.JOB_NAME}/${env.BUILD_ID}/"
+                    }
+                  ]
+                }"""
+
+                def buildInfo = Artifactory.newBuildInfo()
+                // hardcoded to 30 for now, this need follow up for usage on per platform level or across all jobs
+                buildInfo.retention maxBuilds: 30, deleteBuildArtifacts: true
```

Comment:
Can probably use this once its ready (just fyi, not blocking this on that)
https://github.com/eclipse/openj9/pull/2138/files#diff-86d1cc0920536c6d4819089e4772aa71R472
https://github.com/eclipse/openj9/pull/2138/files#diff-a3fca431c8c372cebd1caff040ca6eabR59